build.gradle

```diff
@@ -99,6 +99,13 @@ node {
 processResources {
     dependsOn npm_install
 
+    doFirst {
+        File resourcesDir = sourceSets.main.output.resourcesDir
+        resourcesDir.mkdirs()
+        File versionPropertyFile = new File(resourcesDir, "$VERSION_PROPERTIES_FILE")
+        versionPropertyFile.append("sidebar-sdk-java-version=$currentVersion")
```

Comment:
maybe .append is a bit risky, if the file already exists, we may have duplicated properties
if we use .text instead, it would override the whole content I think